packages/search-in-workspace/src/browser/search-in-workspace-service.ts

```diff
@@ -43,12 +49,16 @@ export class SearchInWorkspaceClientImpl implements SearchInWorkspaceClient {
 
 export type SearchInWorkspaceCallbacks = SearchInWorkspaceClient;
 
+/**
+ * Service to search text in the workspace files.
+ */
+
 /**
  * Service to search text in the workspace files.
  */
 
 @injectable()
-export class SearchInWorkspaceService implements SearchInWorkspaceClient {
+export class SearchInWorkspaceService implements SearchInWorkspaceClient, ISearchInWorkspaceService {
```

Comment:
please remove `ISearchInWorkspaceService` and use `SearchInWorkspaceService` directly, i.e.  `container.get(SearchInWorkspaceService)`

Reply:
Weird somehow I managed to push to the wrong branch name, sorry about that.